book/book.py

```diff
@@ -0,0 +1,39 @@
+import json
+import xml.etree.ElementTree as ET
+
+
+class Book:
+    def __init__(self, title: str, content: str):
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
+        if display_type == "console":
+            print(self.content)
+        elif display_type == "reverse":
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")
+
+    def print_book(self, print_type: str) -> None:
+        if print_type == "console":
+            print(f"Printing the book: {self.title}...")
+            print(self.content)
+        elif print_type == "reverse":
+            print(f"Printing the book in reverse: {self.title}...")
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown print type: {print_type}")
+
+    def serialize(self, serialize_type: str) -> str:
+        if serialize_type == "json":
+            return json.dumps({"title": self.title, "content": self.content})
+        elif serialize_type == "xml":
+            root = ET.Element("book")
+            title = ET.SubElement(root, "title")
+            title.text = self.title
+            content = ET.SubElement(root, "content")
```

Comment:
In case where we talking about SOLID you should adhere SOLID principles, In your case, you could create separate classes for handling serialization, printing, and displaying(and handler with specific fetching properties). Also you should create separate interfaces for different types of handler which will handle somehow data related to book.

And these entities should get all reldated data on main process.